From 5c28782451f680c76f0efbe9e16da2dc70765af7 Mon Sep 17 00:00:00 2001
From: Luke Yue <lukedyue@gmail.com>
Date: Tue, 21 Jul 2020 14:12:38 +0800
Subject: [PATCH] add openEuler

Signed-off-by: Luke Yue <lukedyue@gmail.com>
---
 neofetch | 27 +++++++++++++++++++++++++--
 1 file changed, 25 insertions(+), 2 deletions(-)

diff --git a/neofetch b/neofetch
index 1bcb1a1..98f32be 100755
--- a/neofetch
+++ b/neofetch
@@ -769,7 +769,7 @@ image_source="auto"
 #       LMDE, Lubuntu, Lunar, macos, Mageia, MagpieOS, Mandriva,
 #       Manjaro, Maui, Mer, Minix, LinuxMint, MX_Linux, Namib,
 #       Neptune, NetBSD, Netrunner, Nitrux, NixOS, Nurunner,
-#       NuTyX, OBRevenge, OpenBSD, OpenIndiana, OpenMandriva,
+#       NuTyX, OBRevenge, OpenBSD, openEuler, OpenIndiana, OpenMandriva,
 #       OpenWrt, osmc, Oracle, PacBSD, Parabola, Pardus, Parrot,
 #       Parsix, TrueOS, PCLinuxOS, Peppermint, popos, Porteus,
 #       PostMarketOS, Proxmox, Puppy, PureOS, Qubes, Radix, Raspbian,
@@ -4828,7 +4828,7 @@ ASCII:
                                 LMDE, Lubuntu, Lunar, macos, Mageia, MagpieOS, Mandriva,
                                 Manjaro, Maui, Mer, Minix, LinuxMint, MX_Linux, Namib,
                                 Neptune, NetBSD, Netrunner, Nitrux, NixOS, Nurunner,
-                                NuTyX, OBRevenge, OpenBSD, OpenIndiana, OpenMandriva,
+                                NuTyX, OBRevenge, OpenBSD, openEuler, OpenIndiana, OpenMandriva,
                                 OpenWrt, osmc, Oracle, PacBSD, Parabola, Pardus, Parrot,
                                 Parsix, TrueOS, PCLinuxOS, Peppermint, popos, Porteus,
                                 PostMarketOS, Proxmox, Puppy, PureOS, Qubes, Radix, Raspbian,
@@ -8118,6 +8118,29 @@ ${c1} `-|.'   /_.          ${c4}\_|  ${c1} F
 EOF
         ;;
 
+        "openEuler"*)
+            set_colors 4 7 1
+            read -rd '' ascii_data <<'EOF'
+${c1}
+                       (#####
+                     (((########  #####
+                    (((        ##########    __...__
+             ((((((((           #######    /((((((###\
+           (((((((((((   .......           \(((((####/
+          ((((((    ((((#########            *******
+    %((((((#          ((########
+ /////(((((              ###
+/////(((((((#   (((&
+         (((((((((((((
+          ((((((((((((
+           (((((((((     ((((((###
+                       /((((((######
+                      //((((((######
+                       /((((((#####
+                        *********/
+EOF
+        ;;
+
         "OpenIndiana"*)
             set_colors 4 7 1
             read -rd '' ascii_data <<'EOF'
-- 
2.27.0

