diff --git a/src/parserc.c b/src/parserc.c
index e271ebb63779feba1894b803ab1cdd81bcab4e6a..ffd5a2b43de29c064d16d9c52dca365476178c2c 100644
--- a/src/parserc.c
+++ b/src/parserc.c
@@ -211,7 +211,7 @@ setStringValue (const gchar * lvalue, const gchar *value, Settings *rc)
 {
     GValue tmp_val = {0, };
     g_value_init(&tmp_val, G_TYPE_STRING);
-    g_value_set_static_string(&tmp_val, value);
+    g_value_set_string(&tmp_val, value);
     return setGValue (lvalue, &tmp_val, rc);
 }
 
